xc7/litex_demo/generate.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+import os
+import argparse
+
+from litex.boards.targets.arty import _CRG
+from liteeth.phy.mii import LiteEthPHYMII
+from litex.soc.cores.led import LedChaser
+
+from litex.boards.platforms import arty
+from litex.soc.integration.soc_core import SoCCore
+from litex.soc.integration.builder import Builder
+from litex.soc.integration.soc_core import soc_core_args, soc_core_argdict
+from litex.build.xilinx.vivado import vivado_build_args, vivado_build_argdict
+
+# TODO: Add SDRAM support for SymbiFlow Toolchain:
```

Comment:
Can you please link to an issue that will cover the work to fix this TODO?

Reply:
fixed with https://github.com/enjoy-digital/litex/pull/704